### ISSUE TITLE
Small update of the spec for joining or splitting trips

### ISSUE THREAD

**Issue by pietercolpaert** [openplannerteam/itinero-transit#13]
https://github.com/openplannerteam/planner.js/issues/18

Not yet in  a public knowledge graph, yet know that the spec slightly updated to be able to support this use case!

**Comment by xivk** [comment]
@pietervdvn Has this been taken into account? Can we close this or not?

**Comment by pietercolpaert** [comment]
No test data available yet for this

**Comment by pietervdvn** [comment]
No, I haven't taken this into account, because I'm not convinced of the spec.

If two [Desiro-Trains are linked together](https://i.ytimg.com/vi/XD25O7hOozU/maxresdefault.jpg), it is _not_ possible to walk from the front to the end - see picture. So, consider the following situation:

- Train A departs in ADep
- Train B departs in BDep
- They merge in some location
- They split in some later location
- Train A continues till AArr
- Train B continues till BArr

A traveller which goes from ADep till BArr will _have to leave_ the train at some location to mount the other train. In other words, I'd model these trains as _two completely seperate trips_, which coincidentally are on the same places at the same times for a part of the track.

**Comment by pietercolpaert** [comment]
Please suggest in the spec how you would model this then.

Also, in cases where they are actually merged it appears you do agree with
the spec.

> --
+32486747122
